tests/h/accounts/views_test.py

```diff
@@ -994,6 +1143,18 @@ def pyramid_request(self, pyramid_request, fake_db_session):
         return pyramid_request
 
 
+class RedirectMatcher(object):
+    """Matches any HTTPFound redirect to the given URL."""
```

Comment:
The name doesn't match the behaviour here. Either this should match all redirects (`isinstance(other, httpexceptions.HTTPRedirection`) or it should be called something like `redirect_302_to`.

For consistency with other matcher objects I'd suggest giving it a `lower_underscored` name.